src/dotty/tools/dotc/transform/PostTyper.scala

```diff
@@ -274,6 +274,18 @@ class PostTyper extends MacroTransform with IdentityDenotTransformer  { thisTran
               case tpe => tpe
             }
           )
+        case Import(expr, selectors) =>
+          val exprTpe = expr.tpe
+          def notExists(name: Name): Boolean =
+            name != nme.WILDCARD && !exprTpe.member(name).exists
```

Comment:
I also made it encode the name. This was needed for example in `scala.collections.immutable.::`.